app/assets/javascripts/application.js

```diff
@@ -0,0 +1,24 @@
+//= require smart-answers
+//= require helpers
+//= require start-button-ab-test-july-2016
+
+$(document).ready(function() {
+  $('#current-error').focus();
+});
+
+function linkToTemplatesOnGithub() {
```

Comment:
@selfthinker I think this function should move as well to the helpers.js. This function is principally helps highlight content and provide a link to the template on github.